### ISSUE TITLE
Webpack2 import

### ISSUE THREAD

**Issue by balajmarius** [lukehedger/sip#12]
I'm trying to use `Sip.css` with `webpack 2 / react-create-app` but the result of the import is a string.

```js
import Sip from 'sip.css'

console.log(Sip) -> /static/media/sip.5c5e6212.mjs
console.log(typeof Sip) -> string
```
I am doing something wrong?

**Comment by lukehedger** [comment]
@balajmarius interesting - looks like it thinks 'sip.css' references a CSS file! Must be something to do with CRA webpack config, as works with a [simple config](https://github.com/lukehedger/sip/blob/master/examples/styled-components/webpack.config.js).

**Comment by balajmarius** [comment]
You are right, CRA uses `css-loader` and `sip.css` is interpreted as a CSS file.

**Comment by lukehedger** [comment]
@balajmarius I guess if Sip is being used then so will a CSS-in-JS library, which means `css-loader` will not be needed? CRA users could then just run `npm run eject` and remove from Webpack config.

It is interesting that [CSS-in-JS isn't the default](https://github.com/facebookincubator/create-react-app#limitations) for CRA but I imagine they wanted to keep it as simple as possible!
